docs/platform/concepts/permissions.md

```diff
@@ -47,6 +47,10 @@ and services within it.
 You can grant the following roles and permissions to principals. Roles and permissions
 granted at this level apply to the project and all services within it.
 
+These permissions apply to the
+[project API endpoint](https://api.aiven.io/v1/organization/{organization_id}/projects)
```

Comment:
Not sure this link works as intended. It leads to URL `https://api.aiven.io/v1/organization/%7Borganization_id%7D/projects`.

Reply:
Ah, I pasted the wrong thing - thanks for catching this!